Create 2 tctl topics (#687)
temporalio/documentation: docs/content/what-is-tctl.md

---
id: what-is-tctl
title: What is tctl?

> description: The Temporal CLI (tctl) is a command-line tool that you can use to perform various tasks on a Temporal Server.

Reviewer: ```suggestion
description: The Temporal CLI (tctl) is a command-line tool that you can use to interact with a Temporal Cluster.
```

tags:

>   - operation-guide
> ---
> 

Reviewer: ```suggestion
  - operation-guide
  - tctl
---

```
